CPS-????/README.md

```diff
@@ -0,0 +1,137 @@
+
+**Library Maintainers**: Serialization library authors currently must support multiple encoding strategies for compatibility. With a standard, they can focus on a single canonical implementation, reducing maintenance burden and improving deserialization reliability.
+
+## Goals
+
+<!-- A list of goals and non-goals a project is pursuing, ranked by importance. These goals should help understand the design space for the solution and what the underlying project is ultimately trying to achieve.
+
+Goals may also contain requirements for the project. For example, they may include anything from a deadline to a budget (in terms of complexity or time) to security concerns.
+
+Finally, goals may also serve as evaluation metrics to assess how good a proposed solution is. -->
+
+### Primary Goals
+
+1. **Establish a canonical CBOR standard**: A CIP that specifies deterministic encoding rules for all Cardano transaction and on-chain data structures, with clear guiding principles for choosing between encoding alternatives.
+
+2. **Achieve ecosystem adoption**: Widespread implementation across major serialization libraries (cardano-serialization-lib, pycardano, Lucid, Aiken, etc.) and wallets (Nami, Eternl, Lace, Yoroi, etc.), ensuring cross-tool interoperability.
+
+3. **Provide implementation guidance**: Comprehensive documentation including test vectors, reference implementations or validation tools, and migration paths for existing tooling.
+
+### Optional Goals
+
+4. **Ledger-level enforcement**: If community consensus supports it, implement validation rules in the ledger to guarantee compliance (requires hardfork and backward compatibility strategy).
+
+### Success Criteria
+
+This CPS is successfully resolved when:
+
+- A canonical CBOR serialization CIP reaches "Active" status with clear specifications
+- At least 80% of major libraries and wallets demonstrate compliance
+- Serialization-related issues in community support channels decrease measurably
+- Cross-tool transaction building becomes reliably predictable
+
+### Requirements for Solutions
+
+A good solution must:
+
+- **Technical clarity**: Unambiguous encoding rules for all covered data structures
+- **Guiding principles**: Clear rationale for choosing specific encodings (efficiency, simplicity, adoption)
+- **Comprehensive scope**: Address transactions, scripts, datums, redeemers, and specify what is out-of-scope
+- **Path to Active**: Detailed adoption strategy including timeline, stakeholder coordination, and migration tooling
+- **Evolution mechanism**: Process for handling future hardforks and new ledger types
+- **Verification**: Test vectors or validation tools to verify implementation compliance
+
+## Open Questions
+
+### What are the guiding principles for choosing the canonical form?
```

Comment:
If a custom canonical CBOR standard is to be designed then there is much higher chance that it will be implemented correctly by all of the tools. Grabbing an existing standard and accepting any potential drawback it could have (eg. non-optimal size) would be a safer bet IMHO.